hack/verify-test-code.sh

```diff
@@ -21,7 +21,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${KUBE_ROOT}"
 
 # NOTE: This checks e2e test code without the e2e framework which contains Expect().To(HaveOccurred())
-mapfile -t all_e2e_files < <(find test/e2e -name '*.go' | grep -v 'test/e2e/framework/')
+mapfile -t all_e2e_files < <(find test/e2e{,_node,_kubeadm} -name '*.go' | grep -v 'test/e2e/framework/')
```

Comment:
NOTE: see previous discussions in this repo: we don't use mapfile because it doesn't exist on macOS. there are examples of alternatives eleswhere in hack/ such as the verify-shellcheck script (read loop)

Reply:
this particular change looks fine, but the original implementation probably should not have introduced mapfile usage

Reply:
@BenTheElder I see, thanks for your comment. I will follow the mapfile issue with another PR with some digging.

Reply:
There is a tool inside `hack/lib/util.sh` called `kube::util::read-array` that works the same way as `mapfile -t`, and works in bash3 :smile: @oomichi @BenTheElder But yeah, that can be fixed in another PR.

https://github.com/kubernetes/kubernetes/blob/b3a73f761f5b67f783d40a8b89dce47c5a015909/hack/lib/util.sh#L715

Reply:
@oomichi @odinuge thanks! that's exactly the thing to do, let's do that in another PR. feel free to ping me on that one for approval :sweat_smile:

Reply:
https://github.com/kubernetes/kubernetes/pull/79186 is the follow-up PR.